backbone/src/main/java/org/researchstack/backbone/step/InstructionStep.java

```diff
@@ -70,7 +70,7 @@
     private SubmitBarNegativeActionSkipRule submitBarSkipRule;
 
     /* Default constructor needed for serilization/deserialization of object */
-    InstructionStep() {
+    public InstructionStep() {
```

Comment:
Need to create a custom step in the MPower project that extended InstructionStep.